tests/scripts/thread-cert/Cert_5_6_09_NetworkDataForwarding.py

```diff
@@ -96,19 +104,21 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[SED].get_state(), 'child')
 
-        self.nodes[LEADER].add_prefix('2001:2:0:1::/64', 'paros', 'med')
-        self.nodes[LEADER].add_route('2001:2:0:2::/64', 'med')
+        self.collect_rloc16s()
+        self.collect_ipaddrs()
+        self.nodes[LEADER].add_prefix('2001:2:0:1::/64', 'aros', 'med')
+        self.nodes[LEADER].add_route('2001:2:0:2::/64', 's', 'med')
         self.nodes[LEADER].register_netdata()
 
         # Set lowpan context of sniffer
         self.simulator.set_lowpan_context(1, '2001:2:0:1::/64')
 
         self.simulator.go(10)
 
-        self.nodes[ROUTER2].add_prefix('2001:2:0:1::/64', 'paros', 'low')
-        self.nodes[ROUTER2].add_route('2001:2:0:2::/64', 'high')
+        self.nodes[ROUTER2].add_prefix('2001:2:0:1::/64', 'aos', 'med')
+        self.nodes[ROUTER2].add_route('2001:2:0:2::/64', 's', 'high')
```

Comment:
```suggestion
        self.nodes[ROUTER2].add_route('2001:2:0:2::/64', stable=True, prf='high')
```
should use `True/False` for `stable` argument?

Reply:
added